src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -393,9 +391,28 @@ public static String getPathWithoutBucket(String path) {
         final String[] split = gcsUrl.split("/");
         final String BUCKET = split[2];
         final String pathWithoutBucket = String.join("/", Arrays.copyOfRange(split, 3, split.length));
-        return CloudStorageFileSystem.forBucket(BUCKET).getPath(pathWithoutBucket);
+        CloudStorageConfiguration cloudConfig = CloudStorageConfiguration.builder()
+            // if the channel errors out, re-open up to this many times
+            .maxChannelReopens(3)
+            .build();
+        StorageOptions sopt = setGenerousTimeouts(StorageOptions.newBuilder()).build();
+        return CloudStorageFileSystem.forBucket(BUCKET, cloudConfig, sopt).getPath(pathWithoutBucket);
     }
 
+    private static StorageOptions.Builder setGenerousTimeouts(StorageOptions.Builder builder) {
+        return builder
+            .setTransportOptions(HttpTransportOptions.newBuilder()
+                .setConnectTimeout(60000)
+                .setReadTimeout(60000)
+                .build())
+            .setRetryParams(RetryParams.newBuilder()
+                .setRetryMaxAttempts(10)
+                .setRetryMinAttempts(6)
+                .setMaxRetryDelayMillis(30000)
+                .setTotalRetryPeriodMillis(120000)
+                .setInitialRetryDelayMillis(250)
+                .build());
+    }
     // TODO(jpmartin): uncomment once shaded.cloud_nio.com.google.auth.Credentials is visible.
```

Comment:
Actually there was a new release just two days ago. Updating to the newest gcloud should probably be a separate PR, though.